src/main/drivers/sdcard_spi.c

```diff
@@ -557,7 +589,7 @@ static void sdcardSpi_init(const sdcardConfig_t *config, const spiPinConfig_t *s
     // Note that this does not release the CS at the end of the transaction
```

Comment:
This comment does not reflect changed code

Reply:
Replaced with `// Note that CS is not asserted for this transaction`